doc/classes/JSONRPC.xml

```diff
@@ -11,42 +13,55 @@
 			<return type="Dictionary">
 			</return>
 			<argument index="0" name="method" type="String">
+    			Name of the method being called.
```

Comment:
There are indentation issues where spaces and tabs are mixed. All indentation should be done with tabs.